examples/traces/features/batch_exporting.php

```diff
@@ -12,14 +13,15 @@
 echo 'Starting ConsoleSpanExporter with BatchSpanProcessor' . PHP_EOL;
 echo sprintf('Sending batches every %dms and on shutdown', $delayMillis) . PHP_EOL;
 
-$tracerProvider =  new TracerProvider(
+$tracerProvider = new TracerProvider(
     new BatchSpanProcessor(
         new ConsoleSpanExporter(),
         null,
         2048, //max spans to queue before sending to exporter
         $delayMillis, //batch delay milliseconds
     )
 );
+ShutdownHandler::register([$tracerProvider, 'shutdown']);
```

Comment:
I would prefer using `$tracerProvider->shutdown()` at the end of the script for the examples.

Reply:
done.